### ISSUE TITLE
Add Explainable AI

### ISSUE THREAD

**Issue by matuskalas** [edamontology/edam-bioimaging#24]
Add a topic related to Explainable AI / AI explainability / ... Good terms and definition have to be found. Wikipedia searching to start with... Please add it in WebProtégé in form of a comment thread under the most appropriate "parent" concept (live dev version, see README)

**Comment by AnkitaxPriya** [comment]
I would like to work on it @matuskalas 

Do we need to add just the meanings of *Explainable AI* here in this thread or add them in WebProtégé directly? The mention of WebProtégé isn't very prominent in the `README.md` file. Also, what is the procedure to add meanings of *Explainable AI* in WebProtégé?
Kindly add a more detailed description to this issue @matuskalas

**Comment by Jannathfatima** [comment]
I would like to work on it @matuskalas

**Comment by Jannathfatima** [comment]
@matuskalas please can you explain what to do in this , I am new to git and I want to contribute in Machine learning please help me find how to contribute in ML

**Comment by anita0707** [comment]
@matuskalas  interested in this issue but need more details to work with.I have searched on the meaning and examples of explainable AI.

**Comment by matuskalas** [comment]
Thanks for your interest👍🏽 I'm assigning this to all 3 of you. And a couple of more people can join...

The details of how to contribute to EDAM-bioimaging are documented in the README and CONTRIBUTING (it's the same text as https://github.com/edamontology/edam-bioimaging#contributing).

Please navigate to the AI topic (https://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Classes?selection=Class(%3Chttp://edamontology.org/topic______Artificial_intelligence%3E)). You need to register at WebProtégé.

The first one of you who's there, please "Start a thread" in "Comments: Artificial intelligence" window (done by @AnkitaxPriya). The next ones, please contribute to **the same** thread. Add notes related to **Explainable artificial intelligence**:
(_N.B._: It's not necessary that everyone adds all of the following. This will be a team work)
- Short, 1-sentence definition
- One (or more) link(s) to Wikipedia articles
- Any additional comments if needed
- If there are more related terms (_e.g._ AI explainability) or more related concepts (in EDAM-bioimaging or missing), please note those also in your comments
- Please note also any open questions, things that need to be discussed further or clarified, _etc._

In case WebProtégé does not work for you, you can do the same at the NCBO BioPortal (also documented in README#contributing). Necessary to register to be able to add Notes/suggestions.

**Comment by ZM202** [comment]
@matuskalas Room for 1 more to work on this? I haven't done with Al before. I want to give this a try. Thanks.

**Comment by AnkitaxPriya** [comment]
@matuskalas I have added a thread here, please have a look if it is right. 
![image](https://user-images.githubusercontent.com/44089458/115139012-3bbb7900-a04d-11eb-822c-bf8f2512059e.png)

@Jannathfatima, @anita0707, and @ZM202 you all may continue to contribute in the same thread, as suggested by @matuskalas in the above comments.

**Comment by AnkitaxPriya** [comment]
@matuskalas Can we add a thread that explains how the Explainable AI model works (in detail)? 

Also, where are we supposed to report the updates we make in WebProtégé?

**Comment by ZM202** [comment]
Hi @AnkitaxPriya I'm not seeing the thread you got there when I created an acct. Am I missing something? Thanks.

**Comment by matuskalas** [comment]
Thank you @AnkitaxPriya for starting the thread! 🙌🏽 Others please contribute to the same thread of messages by clicking on the 'Reply' (you might need to make the subwindow wider to see it in the last comment).

> @matuskalas Can we add a thread that explains how the Explainable AI model works (in detail)?

This would be out of scope of EDAM to go into so much detail.

> Also, where are we supposed to report the updates we make in WebProtégé?

Please record them either as contribution to an issue (this one in this case), or even better a contribution to a concept on WebProtégé (in this case https://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Classes?selection=Class(%3Chttp://edamontology.org/topic______Artificial_intelligence%3E)), if that works on the Outreachy website.

**Comment by ZM202** [comment]
Hi @matuskalas 

I input the information as a reply as mentioned above in webprotege. Please let me know if you're ok with the format in terms of reference citations, etc. I don't know how to change the username. I had it as z2021. I probably should have kept it the same username as here to avoid confusion.

Also, just wondering what is the end goal for this task? Are we creating a page for the explanation etc...Do you need us to work on this further on the next step for this? If so, could you please give us instruction on what you would like us to do. Thanks.

**Comment by AnkitaxPriya** [comment]
@matuskalas Got it! 
Could you please let us know if it is necessary to mention Wikipedia article's link only or we may add other resources as well?

**Comment by Lukong123** [comment]
> Thanks for your interest👍🏽 I'm assigning this to all 3 of you. And a couple of more people can join...
> 
> The details of how to contribute to EDAM-bioimaging are documented in the README and CONTRIBUTING (it's the same text as https://github.com/edamontology/edam-bioimaging#contributing).
> 
> Please navigate to the AI topic (https://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Classes?selection=Class(%3Chttp://edamontology.org/topic______Artificial_intelligence%3E)). You need to register at WebProtégé.
> 
> The first one of you who's there, please "Start a thread" in "Comments: Artificial intelligence" window (done by @AnkitaxPriya). The next ones, please contribute to **the same** thread. Add notes related to **Explainable artificial intelligence**:
> (_N.B._: It's not necessary that everyone adds all of the following. This will be a team work)
> 
>     * Short, 1-sentence definition
> 
>     * One (or more) link(s) to Wikipedia articles
> 
>     * Any additional comments if needed
> 
>     * If there are more related terms (_e.g._ AI explainability) or more related concepts (in EDAM-bioimaging or missing), please note those also in your comments
> 
>     * Please note also any open questions, things that need to be discussed further or clarified, _etc._
> 
> 
> In case WebProtégé does not work for you, you can do the same at the NCBO BioPortal (also documented in README#contributing). Necessary to register to be able to add Notes/suggestions.

Please is there room for one more?

**Comment by anita0707** [comment]
Hi @matuskalas  ,I have inputted some information in the webprotege .Please vet for any recommendations.
Please what's next?.My username on the protege is o.anita.

**Comment by sarkar-ly** [comment]
I tried my hand on web protege. Can I actively participate in this?

**Comment by sarkar-ly** [comment]
@matus kalas what URL should be used to record this contribution
https://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Comments?selection=Class(%3Chttp://edamontology.org/topic______Artificial_intelligence%3E) would this be okay?

**Comment by matuskalas** [comment]
> what URL should be used to record this contribution https://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Comments?selection=Class(%3Chttp://edamontology.org/topic______Artificial_intelligence%3E) would this be okay?

Yes absolutely, this is the perfect URL for recording your contributions here! 👍🏽👍🏽
Thank you @sarkar-ly for the heads-up and solution!🙏🏽

**Comment by sarkar-ly** [comment]
Thanks for the response 😊
